src/Models/ProductModel.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+class ProductModel
+{
+
+    public static function getProductsByCategoryId(PDO $db, int $category_id) : array
```

Comment:
```suggestion
    public static function getProductsByCategoryId(PDO $db, int $category_id): array
```
Inconsistent with your other return hints